lib/api/auth.ts

```diff
@@ -125,6 +127,11 @@ type LoginPayload = {
   password: string
 }
 
+type LoginMfaPayload = {
+  firstPassToken: string,
+  mfaToken?: string
```

Comment:
This property should be required, not with a `?`. We should not be able to send an empty token

Reply:
Yeah this was an oversight from when I used this type for more, but then rfactoried it out. Fixing